CONTRIBUTING.md

```diff
@@ -1,7 +1,31 @@
-Code contributitions are to be done by requesting pull requests. EOSDIS members will then review the contribution and include it if posible.
+# Contributing
 
-Guidlines
+Thanks for considering a contribution to this project!
 
-* Code must include tests
-* Code must be documented
-* Code must follow style guidlines for the langage it is written in
+## Making Changes
+
+To allow us to incorporate your changes, please use the following process:
+
+1. Fork this repository to your personal account.
+2. Create a branch and make your changes, including tests to verify your change
+3. Test the changes locally/in your personal fork.
+4. Submit a pull request to open a discussion about your proposed changes.
+5. The maintainers will talk with you about it and decide to merge or request additional changes.
+
+## Commits
+
+Our ticketing and CI/CD tools are configured to sync statuses amongst each
+other. Commits play an important role in this process. Please start all commits
+with the CMR ticket number associated with your feature, task, or bug. All
+commit messages should follow the format "CMR-XXXX: [Your commit message here]"
+
+## Disclaimer
+
+The Earthdata Search develoment team will review all pull requests submitted.
```

Comment:
don't you mean CMR team? and you have a misspelling

Reply:
your looking at an old copy